pkg/crds/controllers/batch/batchjob_controller.go

```diff
@@ -56,7 +56,7 @@ type BatchJobReconciler struct {
 // +kubebuilder:rbac:groups=batch.cortex.dev,resources=batchjobs/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=batch.cortex.dev,resources=batchjobs/finalizers,verbs=update
 // +kubebuilder:rbac:groups=batch,resources=jobs,verbs=get;list;watch;create;update;patch
-// +kubebuilder:rbac:groups="",resources=pods,verbs=get;list;watch
+// +kubebuilder:rbac:groups="",resources=pods;configmaps,verbs=get;list;watch
```

Comment:
This wont work, you need to add an extra line for configmaps with more permissions (at least create, get, list, watch)

Reply:
Are you referring to the process of creating a configmap, because that thing currently works. What would this new line look like?

Reply:
Would it be like

```
// +kubebuilder:rbac:groups="",resources=configmaps,verbs=create;get;list;watch
```

?

Reply:
Yes, exactly